cfme/tests/control/test_alerts.py

```diff
@@ -108,10 +108,14 @@ def vddk_url(provider):
         major = str(provider.version)
         minor = "0"
     vddk_version = "v{}_{}".format(major, minor)
-    try:
-        return cfme_data.get("basic_info").get("vddk_url").get(vddk_version)
-    except AttributeError:
+    # cf. BZ 1651702 vddk_version 6_7 does not currently work with CFME, so use v6_5
+    if vddk_version == "v6_7":
```

Comment:
Required:
I would like to use `BZ(....).blocks`. So, If BZ fixed test should be run with `v6_7`. What do you say?

Reply:
that's fine with me

Reply:
also, the url for vddk v6_7 is not currently in the cfme_data.yaml file -- but can be added when this bug is fixed.